generators/generators-commons/src/main/java/uk/gov/justice/services/generators/commons/helper/MessagingResourceUri.java

```diff
@@ -26,7 +26,16 @@ public String destinationName() {
      * @return capitalised uri string without delimiters
      */
     public String toCapitalisedString() {
```

Comment:
This method probably needs to be renamed if its only being used to generate a class name. Maybe split out the logic that generates a name from the logic that ensures its a valid class name?